packages/frontend/src/components/message/MessageList.tsx

```diff
@@ -37,8 +37,8 @@ import { markChatAsSeen } from '../../backend/chat'
 type ChatTypes =
   | C.DC_CHAT_TYPE_SINGLE
   | C.DC_CHAT_TYPE_GROUP
-  | C.DC_CHAT_TYPE_BROADCAST
-  | C.DC_CHAT_TYPE_MAILINGLIST
+  | C.DC_CHAT_TYPE_IN_BROADCAST
+  | C.DC_CHAT_TYPE_OUT_BROADCAST
```

Comment:
So, `DC_CHAT_TYPE_BROADCAST` was seemingly just renamed to `DC_CHAT_TYPE_OUT_BROADCAST`, but what about `DC_CHAT_TYPE_MAILINGLIST`? It seems to be still a thing in core, it doesn't seem like it's just a one-to-one replacement. `DC_CHAT_TYPE_IN_BROADCAST` has a different ID.

Do you know a good resource to learn about this?

Reply:
So if a chat is `DC_CHAT_TYPE_MAILINGLIST`, then we're supposed to e.g. show "Mailing list" in the "View Chat" dialog, and not "Channel"? Am I reading this correctly?

https://github.com/deltachat/deltachat-desktop/issues/5258 says:

> Rename "broadcast list" to "channel" in the UI. There are new stock strings in which broadcast_list is replaced with channel

It's not entirely correct then?

Reply:
> So if a chat is DC_CHAT_TYPE_MAILINGLIST, then we're supposed to e.g. show "Mailing list" in the "View Chat" dialog, and not "Channel"? Am I reading this correctly?

yes. the string "Mailing list" should be shown for DC_CHAT_TYPE_MAILINGLIST where it was shown before, no changes required there.

Reply:
So to summarize: on a non chatmail account, receiving a "normal" mail sent like by some random mail client will be shown as "Mailinglist" - probable other recipients (multiple recipients or CC recipients) will be shown as "Group members"

Maybe we can add that to FAQ now we have a more clear distinction in the UI

Reply:
> on a non chatmail account, receiving a "normal" mail sent like by some random mail client will be shown as "Mailinglist"

no, this is wrong. only "real, classic mailinglists" will become DC_CHAT_TYPE_MAILINGLIST.

"mail sent like by some random mail client" will become DC_CHAT_TYPE_GROUP or DC_CHAT_TYPE_SINGLE with is_encrypted() == false

maybe it was wrong, to bring up "mailinglist" wrt channels alltogether :) (tho https://github.com/deltachat/deltachat-desktop/issues/5258 only says that IN_BROADCAST is similar to MAILINGLIST, not to replace sth :)

Reply:
> no, this is wrong. only "real, classic mailinglists" will become DC_CHAT_TYPE_MAILINGLIST. usually these are newsletters one has subscribed to, or spam.

Ah - ok. So I had some newsletter which was shown as mailinglist that's why I misinterpreted them
We really need a thorough documentation for these different types

Reply:
Yes I saw that, but I was not aware that Mailinglist in DC context is similar to a "usual" mailinglist.

And the documentation should cover all the different behaviours of the different chat types, like why can't I edit the group name here, but not there etc. Especially when we have another chat type channel.

Would be easier for developers and users :-)

But this is not a complaint, just a remark - I will also contribute to docs on that...

Reply:
@r10s  Is that comment correct?

https://github.com/deltachat/deltachat-desktop/pull/5288/files#diff-8360bd603e5389f39fc1f833ed81642507861965ac651d05d33a5a04183de159R332

Reply:
which comment precisely are you referring to? the link opens the while diff of this PR for me :)

Reply:
Maybe you have to wait a bit until the browser processed the deep link?

It's about this comment:

 // We don't allow editing of non encryped groups (email groups)
  // i.e. changing name, avatar or recipients
  // since it cannot be guaranteed that the recipients will adapt
  // these changes (image is not shown at all in MTAs, group name is
  // just the subject and recipients are basically just an email
  // distribution list)

Reply:
yes, the comment makes sense